apso/src/main/scala/eu/shiftforward/apso/json/ExtraJsonProtocol.scala

```diff
@@ -89,7 +91,11 @@ trait ExtraHttpJsonProtocol {
 trait ExtraMiscJsonProtocol {
   implicit object ConfigJsonFormat extends JsonFormat[Config] {
     def write(conf: Config): JsValue = conf.root.render(ConfigRenderOptions.concise()).parseJson
-    def read(json: JsValue): Config = ConfigFactory.parseString(json.toString)
+    def read(json: JsValue): Config = try {
+      ConfigFactory.parseString(json.toString)
+    } catch {
+      case t: Throwable => deserializationError("Could not parse config: " + json, t)
+    }
```

Comment:
Any reason for the explicit `try ... catch` here and using `Try` [here](https://github.com/ShiftForward/apso/pull/7/files#diff-f1cf56ee5fa9ed57f13febbb7187be73R85)?